src/components/game-setup/VolumeButton.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import { MainStoreContext, mainStore } from "../../stores/mainStore";
+import { VolumeButton } from "./VolumeButton";
+
+describe("VolumeButton", () => {
+  it("should toggle the volume button when clicked", () => {
+    const store = mainStore;
```

Comment:
just FYI if there are gonna be other tests then you can move this line right after `describe()`